kolibri/plugins/management/assets/src/vue/user-page/index.vue

```diff
@@ -198,10 +202,29 @@
     },
     $trNameSpace: 'userPage',
     $trs: {
-      noUsersExist: 'No Users Exist.',
-      allUsersFilteredOut: 'No users match the filter.',
+      // input & accessibility labels
+      filterUserType: 'Filter User Type',
+      editAccountInfo: 'Edit Account Information',
+      searchText: 'Search for a user...',
+      // filter select entries
+      allUsers: 'All Users',
+      admins: 'Admins',
```

Comment:
in reference to lines `211` + `217` (and the other 2 singular/plural combos)

I know about that plural syntax outlined in formatJS, but it didn't seem appropriate here since we're only ever doing one or the other, and these are being used as labels rather than grammatically correct sentences. I was thinking about using a [select](http://formatjs.io/guides/message-syntax/#select-format) but that seemed unnecessarily complex. Opinions?